index.html

```diff
@@ -405,39 +515,75 @@ <h3 id="placement-of-mono-ruby">Placement of mono-ruby</h3>
 <section>
 <h3 id="placement-of-group-ruby">Placement of group-ruby</h3>
 
-<p>Group-ruby is placed as follows:</p>
+<p>
+<span class="checkme">
+In this section, placement rules of group-ruby are described as combinations 
+of two groups of characters, 
+one as 'Western characters' which has proportional width 
+and consisted with characters like 
+<a href="https://www.w3.org/TR/jlreq/#cl-24">Grouped numerals (cl-24)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-25">Unit symbols (cl-25)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-26">Western word space (cl-26)</a>,
+and 
+<a href="https://www.w3.org/TR/jlreq/#cl-27">Western characters (cl-27)</a>
+[[JLREQ]], 
+and another as 'Kana characters' which consisted with characters like 
+<a href="https://www.w3.org/TR/jlreq/#cl-15">Hiragana (cl-15)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-26">Katakana (cl-16)</a>, and 
+<a href="https://www.w3.org/TR/jlreq/#cl-19">Ideographic characters (cl-19)</a>
+[[JLREQ]]. 
+For Western characters, strings are read by clusters of multiple characters, 
+adding spacing between characters for justification are wanted to avoid. 
```

Comment:
> adding spacing between characters for justification are wanted to avoid

Maybe change it to something like:

> it is best to avoid increasing spacing between characters for justification

Reply:
let me write as 
> it is desired to avoid adding spacing between characters for justrification

Reply:
s/justrification/justification/

Reply:
thx. fixed by 1cb7206